src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
```

Comment:
Please use secure http (`https`) protocol for your link in action, by using http you are causing this:

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/87986955/fafdb2c6-aba1-43ef-abc8-422c9411f8b8)

to appear on your input on focus

Reply:
Also you may just omit method and action here in this example as you are not sending anything anywhere from this form